### DISCUSSION TITLE
Could it be possible to implement which parts of the keyboard ui adapts to app color??

### DISCUSSION THREAD

**Discussion by imDusht** [florisboard/florisboard#1211] [General]
Hi Dev, right now adapt color to app theme option changes keyboard : background and key : background color, i was wondering if it could be possible to implement that we can choose which parts of keyboard ui adapts to app color, like implement an option which lists keyboard : background, key : foreground (currently it doesn't changes forever white), key : background parts and we can choose one or multiple options between them.

**Comment by imDusht** [comment]
If it could be possible to implement i've more in mind.

**Comment by patrickgold** [comment]
With the current theme engine it is impossible, though that doesn't mean I can't implement such thing in the future. If doing so I'd probably attach it to the theme rework, as that itself requires the structure to change, so adding an attribute for this shouldn't be the problem.

> currently it doesn't changes forever white

That's incorrect, the foreground can also be black. The background color must be bright for that to happen though.

**Reply by imDusht** [reply]
Thanks.
>That's incorrect, the foreground can also be black. The background color must be bright for that to happen though.

Makes sense otherwise key:foreground would be difficult to read. (Though i haven't experienced it just yet). Sorry for my ignorance

**Reply by patrickgold** [reply]
As this is basically a feature request, I'll convert it to one so I can track it. (#1216)
